modules/reactor/src/meshgenerators/PatternedCartesianMeshGenerator.C

```diff
@@ -959,14 +983,16 @@ PatternedCartesianMeshGenerator::generate()
   }
   // Assign customized outer surface boundary id and name
   if (_external_boundary_id > 0)
-    MooseMesh::changeBoundaryId(*out_mesh, OUTER_SIDESET_ID, _external_boundary_id, false);
+    MooseMesh::changeBoundaryId(*out_mesh, stitching_boundary_id, _external_boundary_id, false);
```

Comment:
Same, this only makes sense if `stitching_boundary_id` is the external boundary (and if not we correct it as above)

Reply:
added to doco